src/Compilers/Core/Portable/WellKnownMember.cs

```diff
@@ -511,6 +511,10 @@ internal enum WellKnownMember
 
         System_Runtime_CompilerServices_PreserveBaseOverridesAttribute__ctor,
 
+        System_StringBuilder__AppendString,
```

Comment:
>System_StringBuilder [](start = 8, length = 20)

`System_Test_StringBuilder`

Reply:
`System_Text_StringBuilder`

Finally got a typo on @cston. Think the score is probably 10,000 to 1 :wink: